lib/status.js

```diff
@@ -4,6 +4,8 @@ const WINDOW = Symbol('window');
 const BUCKETS = Symbol('buckets');
 const TIMEOUT = Symbol('timeout');
 const PERCENTILES = Symbol('percentiles');
+const BUCKET_INTERVAL = Symbol('emit-interval');
```

Comment:
```suggestion
const BUCKET_INTERVAL = Symbol('bucket-interval');
```
?

Reply:
i think i was deciding between the names emit and bucket. thanks